### ISSUE TITLE
HTF-0.13.2.4 does not compile

### ISSUE THREAD

**Issue by peti** [skogsbaer/HTF#69]
Citing from https://hydra.nixos.org/build/83019005:

~~~
Test/Framework/QuickCheckWrapper.hs:169:26: error:
    Not in scope: data constructor ‘InsufficientCoverage’
    |
169 |                   Right (InsufficientCoverage { output=msg }) ->
    |                          ^^^^^^^^^^^^^^^^^^^^
~~~~

**Comment by peti** [comment]
I suppose this issue is caused by the latest version of QuickCheck? The build is reporting:

    Dependency QuickCheck >=2.3: using QuickCheck-2.12.6.1

**Comment by mitchellwrosen** [comment]
I've opened #71 to address this

**Comment by skogsbaer** [comment]
#71 has been merged and should fix this issue.

**Comment by skogsbaer** [comment]
Just uploaded 0.13.2.5 to hackage. This version includes #71 and thus fixes bug #69.
